instant/src/main/java/com/adyen/checkout/instant/DefaultInstantPaymentDelegate.kt

```diff
@@ -33,9 +37,16 @@ internal class DefaultInstantPaymentDelegate(
     private val analyticsRepository: AnalyticsRepository,
 ) : InstantPaymentDelegate {
 
-    override val componentStateFlow: Flow<PaymentComponentState<PaymentMethodDetails>> =
+    override val componentStateFlow: StateFlow<PaymentComponentState<PaymentMethodDetails>> =
```

Comment:
Why can't this be a `Flow` anymore?

Reply:
To be able access the value. In the `init` block we read the value and pass it to the `submitChannel`.